### PR TITLE
Reel box

### LINKED ISSUE #24: reel box
Reel foreground, slots for the reels,

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pixiv6/src/model/TGameData.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
+export type TFullUserData = {
```
Comment (hlibSigma) [inline pixiv6/src/model/TGameData.ts]: we already don't needed it. it has been moved to src\server\fruit\service\typing.d.ts

File: pixiv6/src/scenes/SlotMachineScene.ts [hunk 2]
```diff
@@ -1,10 +1,21 @@
+import { GameSize } from "app/model/GameModel";
+
 import BaseScene from "app/scenes/BaseScene";
 import BackgroundControl from "app/controls/BackgroundControl";
 import Resources from "app/pixi/StrictResourcesHelper";
+import ReelBoxControl from "app/controls/SlotMachine/ReelBoxControl";
 
 export default class SlotMachineScene extends BaseScene {
     private readonly background = new BackgroundControl(Resources.getSingleTexture("SLOT-MASHINE_BG"));
+    private readonly reelBox = new ReelBoxControl(5, 16, 1500);
```
Comment (hlibSigma) [inline pixiv6/src/scenes/SlotMachineScene.ts:10]: why 16?

**Review by hlibSigma — DISMISSED** [review]
in general review is ok

File: pixiv6/src/model/TGameData.ts (hunk 1 above)
Comment (ghost) [reply pixiv6/src/model/TGameData.ts]: removed the file

**Review by hlibSigma — APPROVED** [review]
looks good to me

**Pull request merged**